wgpu/Cargo.toml

```diff
@@ -8,7 +8,18 @@ homepage = "https://wgpu.rs/"
 repository = "https://github.com/gfx-rs/wgpu"
```

Comment:
can we also change this to point to v0.10 branch (which will be created)?
people often get confused about this, so it would be nice to have